### PR TITLE
Minor code changes

### PR DESCRIPTION
**Pull request by P3tray**

### CODE REVIEW

**Comment by P3tray** [comment]
i still dont know why a years worth of commits show

**Comment by P3tray** [comment]
stupid github

**Comment by P3tray** [comment]
not my problem

File: MainModule/Server/Commands/Fun.lua [hunk 1]
```diff
@@ -2681,27 +2681,26 @@ return function(Vargs, env)
...
+							q:Destroy()
 						end
-						local spinGryoAttachment: Attachment = service.New("Attachment")
-						local spinGryo: AlignOrientation = service.New("AlignOrientation")
-
-						spinGryoAttachment.Name = "ADONIS_SPIN_GYRO_ATTACHMENT"
-						spinGryoAttachment.Parent = v.Character.HumanoidRootPart
-
-						spinGryo.Name = "ADONIS_SPIN_GYRO"
+						end
```
Comment (ccuser44) [inline MainModule/Server/Commands/Fun.lua:2697]: This code has improper intending

File: MainModule/Server/Commands/Fun.lua [hunk 2]
```diff
@@ -2681,27 +2681,26 @@ return function(Vargs, env)
 			Function = function(plr: Player, args: {string})
 				local scr = Deps.Assets.Spinner:Clone()
 				scr.Name = "SPINNER"
+				local spinGryoAttachment = service.New("Attachment")
+				spinGryoAttachment.Name = "ADONIS_SPIN_GYRO_ATTACHMENT"
+				local spinGryo = service.New("AlignOrientation")
+				spinGryo.Name = "ADONIS_SPIN_GYRO"
+				spinGryo.MaxTorque = math.huge
+				spinGryo.Mode = Enum.OrientationAlignmentMode.OneAttachment
```
Comment (ccuser44) [inline MainModule/Server/Commands/Fun.lua:2689]: Gyro instead of Gryo

**Review by ccuser44 — CHANGES_REQUESTED** [review]

File: MainModule/Server/Commands/Fun.lua (hunk 2 above)
Comment (P3tray) [reply MainModule/Server/Commands/Fun.lua:2689]: :(

File: MainModule/Server/Commands/Fun.lua (hunk 1 above)
Comment (P3tray) [reply MainModule/Server/Commands/Fun.lua:2697]: :(

**Comment by P3tray** [comment]
You made me close my PR :(

**Comment by ccuser44** [comment]
?

**Comment by P3tray** [comment]
> ?

:(

**Comment by WaviestBalloon** [comment]
P3foodtray is upset

**Comment by P3tray** [comment]
> P3foodtray is upset

:(

**Pull request closed without merging**